frontend/public/components/monitoring/targets.tsx

```diff
@@ -289,13 +315,50 @@ type ListProps = {
   targets: Target[];
 };
 
+type WatchErrorAlertProps = {
+  loadError: { code: number; message: string };
+  onClose: () => void;
+  title: string;
+};
+
 const REDUX_ID = 'monitoringTargets';
 
 const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.lastError });
 
+const WatchErrorAlert: React.FC<WatchErrorAlertProps> = ({ loadError, onClose, title }) => {
+  const { t } = useTranslation();
+  const authInfo =
+    'https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html';
+  const authErrorMsg = (
+    <>
+      <p>{loadError.message}</p>
+      <ExternalLink href={authInfo} text={t('Learn more')} />
+    </>
+  );
+
+  return (
+    <>
+      <Alert
+        className="co-alert"
+        title={title}
+        variant="danger"
+        actionClose={<AlertActionCloseButton onClose={onClose} />}
+      >
+        {loadError.code === 403 ? authErrorMsg : loadError.message}
+      </Alert>
+    </>
+  );
+};
+
 const List: React.FC<ListProps> = ({ loaded, loadError, targets }) => {
   const { t } = useTranslation();
 
+  const [showServiceMonitorError, , , hideServiceMonitorError] = useBoolean(true);
+  const [showPodMonitorError, , , hidePodMonitorError] = useBoolean(true);
```

Comment:
I now realize that you could move this into `WatchErrorAlert`, which I think will give exactly the same behavior. In other words, don't pass in the `onClose` prop and instead add something like the following to `WatchErrorAlert`.

```
  const [showError, , , hideError] = useBoolean(true);

  if (!showError) {
    return null;
  }
```

Then the close button becomes `<AlertActionCloseButton onClose={hideError} />`